src/react-integration/hooks/useResource.ts

```diff
@@ -55,17 +58,33 @@ function useManyResources<A extends ResourceArgs<any, any, any>[]>(
       Params extends Readonly<object>,
       Body extends Readonly<object> | void,
       S extends Schema
-    >([select, params]: ResourceArgs<S, Params, Body>) =>
+    >([fetchShape, params]: ResourceArgs<S, Params, Body>) =>
       // eslint-disable-next-line react-hooks/rules-of-hooks
-      useCache(select, params),
+      useCache(fetchShape, params),
+  );
+  const isExpiredList = resourceList.map(([fetchShape, params]) =>
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    useIsExpired(fetchShape, params),
```

Comment:
should probably add some user warnings if the length of resource list ever changes, given that the rules of hooks are being broken